### DISCUSSION TITLE
TSC Agenda Proposals for April 2023 Meeting.

### DISCUSSION THREAD

**Discussion by lgirdwood** [thesofproject/sof#7406] [Announcements]
Now that SOF v2.5 has just been released it's time for TSC to discus and align on current development direction and feature proposals for v2.6 and beyond.

@thesofproject/steering-committee members please add topics for discussion. Usual 1 hour format as before.

Proposals
 1.  v2.6 features and platform integartions.
 2.  Zephyr integration status and plans.
 3. Module API and core improvements.
 4. Pipeline API and core improvements.
 5. Driver status and forward planning.
 6. Porting SOF to arm64 status and plans
 7. others ....

Pick your available time here: https://doodle.com/meeting/participate/id/e9Dg164d

**Comment by lgirdwood** [comment]
@dbaluta are you able to create a calendar option for TSC maintainers to pick a time slot. Thanks !

**Comment by dbaluta** [comment]
@lgirdwood sure, will create a doodle for that. Any suggested weeks when we should try that. We do have the Easter holidays in the 15-16 April. 

@lgirdwood  Let's try the next 2 weeks: Please share this https://doodle.com/meeting/participate/id/e9Dg164d with Intel & community folks!

**Reply by cujomalainey** [reply]
FYI Google won't be able to attend if it is the week of APR 17 as we are in Taiwan and we have full schedules when out there

@dbaluta maybe we should push this out a bit farther?

**Reply by dbaluta** [reply]
@cujomalainey  Does week starting at APR 10 works for you? I think we can have good slots there on Tue or Wed

**Reply by cujomalainey** [reply]
Yes that week is fine or the week of the 24th

**Reply by dbaluta** [reply]
@cujomalainey I also added week of 24th in the doodle. So far next week looks promising too.

**Reply by lgirdwood** [reply]
Week beginning 24 probably best for me, although I'm having issues with the Doodle. I dont see a calendar and just a blue box telling me more dates are available ?

**Reply by lgirdwood** [reply]
@dbaluta do you know if there any permissions on the doodle, I can see the notification but when I click on teh link I can only see a blank Doodle page.

**Reply by cujomalainey** [reply]
I think I may have flashed the device I was using to fill out the doodle, so I cannot edit my availability, anything the week of the 24th works for me

**Comment by bhiregoudar** [comment]
We are available for any day of this month.

**Comment by lgirdwood** [comment]
@thesofproject/steering-committee ok since Doodle not working for everybody- I will send out an invite for week starting 24th tomorrow if no objections - I will aim for 4pm BST - 8am PST.

**Reply by dbaluta** [reply]
@lgirdwood this works for nxp people. we are back from easter vacation.

**Reply by lgirdwood** [reply]
@thesofproject/steering-committee Thanks all - back from vacation today. Invite sent out for 4pm Wednesday 26th (8am PST). Pls reply if you did not get it.

**Reply by bhiregoudar** [reply]
> @thesofproject/steering-committee Thanks all - back from vacation today. Invite sent out for 4pm Wednesday 26th (8am PST). Pls reply if you did not get it.

Didn't get the meeting request

**Reply by lgirdwood** [reply]
@bhiregoudar invite now sent to Maruthi email.
